src/test/java/com/learning/pages/google/GoogleSearchTest.java

```diff
@@ -1,16 +1,24 @@
 package com.learning.pages.google;
 
-import com.google.inject.Inject;
-import com.learning.ApplicationModule;
-import org.testng.annotations.Guice;
+import static org.testng.Assert.assertNotNull;
+
+import com.learning.TestsConfiguration;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.testng.AbstractTestNGSpringContextTests;
 import org.testng.annotations.Test;
 
-@Guice(modules = ApplicationModule.class)
-public class GoogleSearchTest {
+@ContextConfiguration(classes = {TestsConfiguration.class})
+public class GoogleSearchTest extends AbstractTestNGSpringContextTests {
 
-  @Inject
+  @Autowired
   private GoogleSearchPage googleSearchPage;
 
+  @Test
+  void shouldGooglePageNotBeNullWhenAutowired() {
```

Comment:
Not a feature/system test

Reply:
One comment with note "apply to all test cases/all clases" etc. will be enough :)